guides/auth-private-helm-repos.md

```diff
@@ -0,0 +1,78 @@
+# Private Helm/Git Repositories -- Authorization/PAT For HLD to Materialized Pipeline
+
+An operator may want to store their Helm charts in a different repository than
+their application code. In this scenario, the operators Helm chart git
+repository will most likely be private; making the ability to clone it in the
+HLD-to-Materialized pipeline an issue. To facilitate the ability to clone the
+Helm chart repository, we can piggy back off the ability for
+[Fabrikate to consume env variables consuming personal access tokens (PATs)](https://github.com/microsoft/fabrikate/blob/master/docs/auth.md).
+
+## Utilizing The Default PAT (ACCESS_TOKEN_SECRET)
+
+By default, all services created via `spk service create` will be setup to
+consume the environment variable `ACCESS_TOKEN_SECRET` -- The PAT used to clone
+the application repository in the HLD-to-Materialized pipeline. So if the PAT
+used to setup your HLD-to-Materialized pipeline has full read access to the
+Azure DevOps organizations your Helm git repositories are stored, the pipeline
+is all set by default.
+
+## Making The Pipeline Aware of Custom Environment Variables
+
+In order to tell the pipeline that it needs to utilize a PAT stored in a user
+defined environment variable in the HLD-to-Materialized pipeline, the
+environment variable name -- not the PAT itself -- needs to be tracked in the
```

Comment:
How would I get this variable to AzDo? Maybe make a mention of adding the env var to the variable group before hand

Reply:
hld->materialized pipeline doesn't have a variable group, only the predefined variables we scaffold it with. Added a note/link to azure docs on how to add more variables.